specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyEvents.json

```diff
@@ -634,7 +634,7 @@
     }
   },
   "x-ms-paths": {
-    "/{nextLink}?Next paging op for policy events": {
+    "{nextLink}?Next paging op for policy events": {
```

Comment:
@msyyc Do you know why `?Next paging op for policy events` is added after `{nextLink}`? Usually only `{nextLink}` is required.

Reply:
@weidongxu-microsoft I have test it locally and find it has no influence for generated code. So I guess it is just annotation.

Reply:
@najagasi and @iscai-msft were the original committers of the x-ms-paths.  I imagine the path needed to be "unique" between policyStates.json and policyEvents.json to pass some validation but thats just a guess.  They can comment further

Reply:
@pilor you're correct, `x-ms-paths` needs these two to be unique. So we hack the uniqueness by adding "query parameters"